turn_by_turn/structures.py

```diff
@@ -30,14 +30,38 @@ def __getitem__(self, item):  # to access X and Y like one would with a dictiona
         return getattr(self, item)
 
 
+@dataclass
+class SimulationData:
+    """
+    Object holding multidimensional turn-by-turn simulation data in the form of pandas DataFrames.
+    """
+
+    X: pd.DataFrame  # horizontal data
+    PX: pd.DataFrame  # horizontal momentum data
+    Y: pd.DataFrame  # vertical data
+    PY: pd.DataFrame  # vertical momentum data
+    T: pd.DataFrame  # longitudinal data
+    PT: pd.DataFrame  # longitudinal momentum data
+    S: pd.DataFrame  # longitudinal position data
+    E: pd.DataFrame  # energy data
+
+    def fieldnames(self):
```

Comment:
Why not use this, instead of SIMDATA_FIELDS?

Reply:
Sounds good